Hypnotic-Herons/sprint-3
# moved files into folders

## Link to Ticket
Closes

## Description of Proposed Changes
-

## Steps to Test
```
git fetch --all
git checkout mkmd_navbar
code .

```

Outline the steps to test
look at local host 8000 and make sure that you see the Bangazon welcome page




1. python manage.py makemigrations webiste
1. python manage.py migrate
1. python manage.py runserver



## Impacted Areas in Application



## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
